test/tools.py

```diff
@@ -0,0 +1,102 @@
+import os
+import tempfile
+
+import vsc.utils.run
+import vsc.mympingpong.tools
+from vsc.install.testing import TestCase
+
+# mock the one imported in the tools module, not run_simple itself
+runs_orig = vsc.mympingpong.tools.run_simple
+_parse_orig = vsc.mympingpong.tools._parse_hwloc_xml
+
+
+class ToolsTest(TestCase):
+    """Test tools"""
+
+    def setUp(self):
+        """restore/set orignal functions mocked in test_hwlocmap"""
+        vsc.mympingpong.tools.run_simple = runs_orig
+        vsc.mympingpong.tools._parse_hwloc_xml = _parse_orig
+        super(ToolsTest, self).setUp()
+
+    def test_hwlocmap(self):
+        """Test hwlocmap"""
+
+        xmlout = "%s/test.xml.%s" % (tempfile.gettempdir(), os.getpid())
+
+        global called
+        called = {'runs': 0, '_parse': 0}
+
+        def runs(cmd):
```

Comment:
i assumed that mock could do this more cleanly, but still need to look into it.
any idea how to do the `assertEqual(cmd,...` in mock?